src/cmd/cli/command/commands.go

```diff
@@ -469,6 +470,12 @@ var certGenerateCmd = &cobra.Command{
 	},
 }
 
+var SurveyStdio = &terminal.Stdio{
```

Comment:
We're not actually using that type `terminal.Stdio` so I wouldn't depend on it. I actually think we should use `term.DefaultTerm`

Reply:
[`75aaae6` (#917)](https://github.com/DefangLabs/defang/pull/917/commits/75aaae649553fc77983804dee915d99c0b701cec#diff-ee1019c4f435f4b83ecde23a17468a63ff08fdefefe9782550f904423a1d0cedR65)